pkgs/applications/networking/p2p/transmission/4.nix

```diff
@@ -66,6 +69,8 @@ stdenv.mkDerivation (finalAttrs: {
     ] ++ lib.optionals stdenv.isDarwin [
       # Transmission sets this to 10.13 if not explicitly specified, see https://github.com/transmission/transmission/blob/0be7091eb12f4eb55f6690f313ef70a66795ee72/CMakeLists.txt#L7-L16.
       "-DCMAKE_OSX_DEPLOYMENT_TARGET=${stdenv.hostPlatform.darwinMinVersion}"
+      # qt translation generation segfaults
+      "-DENABLE_NLS=OFF"
```

Comment:
not sure if it helps, but i noticed that the qt build doesn't pull in gettext, though when it does it seems a patch is required to utils.h so it will include libintl.h when `__APPLE__` is defined.

Reply:
> not sure if it helps, but i noticed that the qt build doesn't pull in gettext, though when it does it seems a patch is required to utils.h so it will include libintl.h when `__APPLE__` is defined.

@emilytrau this makes sense to me, what do you think? For reference:

https://github.com/transmission/transmission/blob/a6fe2a64aa7eca089f96006cf082a12f0cde937f/libtransmission/utils.h#L34-L36